src/views/HowToPlay/Layout/StyledInstructionsLayout.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const StyledInstructionsLayout = styled.div`
+  margin-left: 3em;
```

Comment:
You can do all 3 margin properties in one line by setting `margin: 0 3em 3em 3em;`. Here's the [MDN doc](https://developer.mozilla.org/en-US/docs/Web/CSS/margin).